src/components/ArticlesSection/ArticlesSection.jsx

```diff
@@ -1,11 +1,24 @@
 import './ArticlesSection.css'
 import ArticlesList from "../ArticlesList/ArticlesList"
+import { fetchArticles } from "../../api/api"
+import { useEffect, useState } from 'react'
 
 const ArticlesSection = () => {
+  const [articles, setArticles] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+  
+  useEffect(() => {
+    setIsLoading(true)
+    fetchArticles()
+      .then(articles => {
+        setArticles(articles)
+        setIsLoading(false)
+      })
+  }, [])
```

Comment:
This whole component isn't really needed as ll the behaviour can be implemented in the ArticlesList component. However, you can still keep it if you like it like this 🙂

Reply:
Thanks :)